src/client/pages/UsersPage/index.tsx

```diff
@@ -110,12 +113,21 @@ class UsersPage extends React.Component<Props, UsersPageState> {
       .then(() => {
         const {users} = this.state;
 
+        this.createAlert(
+          `Successfully updated ${user.account.email}'s ${this.props.event.name} Application`,
```

Comment:
Can you put quotation marks around all the variables in each of the messages (alerts and logs)

Reply:
```Successfully updated "${user.account.email}" "{this.props.event.name}" Application``` ?